test/OpenTelemetry.Resources.AWS.Tests/AWSEC2DetectorTests.cs

```diff
@@ -20,14 +32,14 @@ public void TestExtractResourceAttributes()
         var hostName = "Test host name";
         var resourceAttributes = AWSEC2Detector.ExtractResourceAttributes(sampleEC2IdentityDocumentModel, hostName).ToDictionary(x => x.Key, x => x.Value);
 
-        Assert.Equal("aws", resourceAttributes[AWSSemanticConventions.AttributeCloudProvider]);
-        Assert.Equal("aws_ec2", resourceAttributes[AWSSemanticConventions.AttributeCloudPlatform]);
-        Assert.Equal("Test account id", resourceAttributes[AWSSemanticConventions.AttributeCloudAccountID]);
-        Assert.Equal("Test availability zone", resourceAttributes[AWSSemanticConventions.AttributeCloudAvailabilityZone]);
-        Assert.Equal("Test instance id", resourceAttributes[AWSSemanticConventions.AttributeHostID]);
-        Assert.Equal("Test instance type", resourceAttributes[AWSSemanticConventions.AttributeHostType]);
-        Assert.Equal("Test aws region", resourceAttributes[AWSSemanticConventions.AttributeCloudRegion]);
-        Assert.Equal("Test host name", resourceAttributes[AWSSemanticConventions.AttributeHostName]);
+        Assert.Equal("aws", resourceAttributes[ExpectedSemanticConventions.AttributeCloudProvider]);
+        Assert.Equal("aws_ec2", resourceAttributes[ExpectedSemanticConventions.AttributeCloudPlatform]);
+        Assert.Equal("Test account id", resourceAttributes[ExpectedSemanticConventions.AttributeCloudAccountID]);
+        Assert.Equal("Test availability zone", resourceAttributes[ExpectedSemanticConventions.AttributeCloudAvailabilityZone]);
+        Assert.Equal("Test instance id", resourceAttributes[ExpectedSemanticConventions.AttributeHostID]);
+        Assert.Equal("Test instance type", resourceAttributes[ExpectedSemanticConventions.AttributeHostType]);
+        Assert.Equal("Test aws region", resourceAttributes[ExpectedSemanticConventions.AttributeCloudRegion]);
```

Comment:
can't it get from defined Semantic conventions ?

Reply:
commented elsewhere